bravado/config.py

```diff
@@ -52,6 +53,33 @@ def from_config_dict(config):
         )
 
 
+class RequestConfig(
+    namedtuple(
+        'RequestConfig',
+        [
+            'also_return_response',
+            'force_fallback_result',
+            'response_callbacks',
+            # options used to construct the request params
+            'connect_timeout',
+            'headers',
+            'use_msgpack',
+            'timeout',
+        ]
+    )
+):
+    @staticmethod
+    def from_request_options_dict(request_options, also_return_response_default):
+        request_config = {
+            key: request_options.get(key, REQUEST_OPTIONS_DEFAULTS.get(key))
```

Comment:
As we need to do a backward incompatible change and we're renaming the concept of request_options into request_config do you mind renaming the constant accordingly?